extension/goplugin/schemas_test.go

```diff
@@ -197,6 +197,16 @@ var _ = Describe("Schemas", func() {
 			Expect(&createdResource).To(Equal(returnedResource))
 		})
 
+		It("Should count resources", func() {
+			c, err := testSchema.Count(goext.Filter{}, context)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(c).To(Equal(uint64(0)))
+			Expect(testSchema.CreateRaw(&createdResource, context)).To(Succeed())
+			c, err = testSchema.Count(goext.Filter{}, context)
```

Comment:
for completeness let's add a case when the filter matches only a part of resources (eg. 5 resources in DB, the filter matches 3 of those)